### ISSUE TITLE
'Potential' SW Whirling Rage Bug

### ISSUE THREAD

**Issue by KySaber** [WarEmu/WarBugs#16821]
![1](https://user-images.githubusercontent.com/26847385/81905699-695f9d00-9608-11ea-82c0-dc44a2e62310.png)
![2](https://user-images.githubusercontent.com/26847385/81905703-6a90ca00-9608-11ea-9512-660b2371a048.png)
![3](https://user-images.githubusercontent.com/26847385/81905704-6b296080-9608-11ea-9e57-22b291050e94.png)

Not sure what's happening here, but my spec is generally always taking up to and including 'No Respite' from the SW assault tree, however I first noticed the number changing today. After a suggestion to check if it was the tree affecting damage, I put 75% of my points into assault, then relogged only to find the damage remaining at 1968, vs 2400 with no points in assault prior. I assume it may potentially be some sort of tool tip bug?

**Comment by Kaelang** [comment]
No points in any trees (fresh 40 ding): 
![image](https://user-images.githubusercontent.com/60117106/81925955-2508d880-95d9-11ea-826d-0bc4ac2ac7b0.png)

All points in SW Assault:
![image](https://user-images.githubusercontent.com/60117106/81926050-4e296900-95d9-11ea-9d9a-1e2252dcbc3e.png)

10 Points in SW Assault:
![image](https://user-images.githubusercontent.com/60117106/81926147-70bb8200-95d9-11ea-9f56-48dd52814a77.png)

Unsure what to suggest, but looks to be working fine for me.

**Comment by KySaber** [comment]
After messing around to figure out the reason for the change in damage numbers, I've found zone hopping to cause the number to change:

https://www.youtube.com/watch?v=hPqkbhbLtl4

As shown, the damage on the tool tip changing is reflected in the actual damage the skill puts out  upon use, so this isn't a tool tip issue. After completely gutting my mastery to be 0 in all trees, the damage remains at 2400 while I'm in lower level zones, however, while changing nothing and moving into end zones, the damage drops to 1600. It seems like some sort of 'side' bolstering is in effect in certain zones. Not really sure how to otherwise describe it.

**Comment by SilverWF** [comment]
It increases damage numbers while in the Assault stance

**Comment by KySaber** [comment]
False, as shown in the video and explained, the damage has nothing to do with the stance or tactics, but changes upon zone/spec altering.

**Comment by Kaelang** [comment]
I'll check on dev in a bit and come back to this.

**Comment by Kaelang** [comment]
![image](https://user-images.githubusercontent.com/60117106/82125535-744c3600-979e-11ea-8e2b-0da3d6bbb43f.png)

In Ellyrion.

Something weird going on with this ability. Removing dubious, keeping tag.
